addons/sale_purchase/models/sale_order_line.py

```diff
@@ -244,6 +267,9 @@ def _purchase_service_create(self, quantity=False):
             sale_line_purchase_map[line] |= purchase_line
         return sale_line_purchase_map
 
+
+        return purchase_lines
+
```

Comment:
This is dead code in the first commit, as it is after `return sale_line_purchase_map`
```suggestion
```

Reply:
oupsy, probably a rebase failed